### PR TITLE
Change toPromise to FirstValueFrom

### PR DESCRIPTION
**Pull request by ueberschaersilas-hg**

### CODE REVIEW

**Comment by albrechtflo-hg** [comment]
Currently merge conflicts, please resolve

**Comment by PolatEmre-hg** [comment]
looks good to me :)

**Review by albrechtflo-hg — CHANGES_REQUESTED** [review]
We could (and should) also remove all related `take(1)`s, as `firstValueFrom`, according to the docs, resolves "as soon as the first value arrives", so no need to artificially crop the Observable.

File: ui/src/app/shared/services/topics.service.ts [hunk 1]
```diff
@@ -379,7 +379,7 @@ export class TopicsService {
         };
 
         return () => combineLatest([this.http.get('/api/topics/' + environmentId + '?includeInternal=true'),
-            this.applicationsService.getAvailableApplications(false)]).pipe(map(values => toTopicArray(values))).pipe(take(1));
```
Comment (albrechtflo-hg) [inline ui/src/app/shared/services/topics.service.ts:380]: Must also be kept.

**Review by albrechtflo-hg — CHANGES_REQUESTED** [review]

**Pull request merged**